packages/components/SelectionDropdown.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useRef } from "react";
+import { TouchableOpacity, View, ViewStyle } from "react-native";
+
+import chevronDownSVG from "./../../assets/icons/chevron-down.svg";
+import chevronUpSVG from "./../../assets/icons/chevron-up.svg";
+import { BrandText } from "./BrandText";
+import { SVG } from "./SVG";
+import { Box } from "./boxes/Box";
+import { TertiaryBox } from "./boxes/TertiaryBox";
+import { useDropdowns } from "../context/DropdownsProvider";
+import {
+  neutral17,
+  neutral33,
+  neutral77,
+  neutralA3,
+  secondaryColor,
+} from "../utils/style/colors";
+import { fontSemibold13, fontSemibold14 } from "../utils/style/fonts";
+import { layout } from "../utils/style/layout";
```

Comment:
Just add a breakline between imports and Props definiton

Reply:
Okay. I will fix it.